cheroot/ssl/__init__.py

```diff
@@ -17,12 +17,14 @@ class Adapter(object):
     """
 
     @abstractmethod
-    def __init__(self, certificate, private_key, certificate_chain=None, ciphers=None):
+    def __init__(self, certificate, private_key, certificate_chain=None, ciphers=None,
+                 verify_mode=None):
```

Comment:
You don't have to have 80 chars wrap: this project's linters are configured to allow up to 120-chars lines. We don't work in 80-char terminals nowadays :)